### PR TITLE
GCP lego for removing a role from a user

### PR DESCRIPTION
**Pull request by SaxenaAnushka102**
## Description
Created a GCP lego for removing a role from a user.

### Testing
Tested lego on [Unskript sandbox](https://us.app.unskript.io/)

### CODE REVIEW

**Comment by SaxenaAnushka102** [comment]
Hello @shloka-bhalgat-unskript! I've submitted a PR. Please review it.

File: GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        :type member: string
+        :param member: user's id to be removed.
+
+        :rtype: confirmation of removal of role."""
+
+    service = discovery.build('iam', 'v1', credentials=credentials)
+    # The resource for which the policy is being requested.
+    resource = 'projects/my-project/serviceAccounts/my-service-account'
+    # TODO: Update placeholder value.
```
Comment (jayasimha-raghavan-unskript) [inline GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py:37]: Does this mean the resource path should be parameterized? meaning, should it be taken as an input parameter to this Lego? Also should it be part of the InputClass?

File: GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    request = service.projects().serviceAccounts().getIamPolicy(resource=resource)
+    response = request.execute()
+    binding = next(b for b in policy["bindings"] if b["role"] == role)
+    if "members" in binding and member in binding["members"]:
+        binding["members"].remove(member)
+    print(binding)
```
Comment (jayasimha-raghavan-unskript) [inline GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py:43]: We can move the Print functions to `lego_printer` routine. Check the other AWS Legos that implements lego printer.

**Review by jayasimha-raghavan-unskript — CHANGES_REQUESTED** [review]
Thank you @SaxenaAnushka102 Could you please take care of the review comments.

File: GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py (hunk 1 above)
Comment (SaxenaAnushka102) [reply GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py:37]: Though the resource path is implicitly declared here, we can take it as a parameter from the user by including it in the input class. Shall I do that?

File: GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py (hunk 2 above)
Comment (SaxenaAnushka102) [reply GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py:43]: Okay, I'll research more about the lego_printer routine & let you know what improvements I can make. Thanks for the suggestion!

File: GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py (hunk 1 above)
Comment (dougsillars) [reply GCP/legos/gcp_remove_user_role/gcp_remove_user_role.py:37]: Hi @SaxenaAnushka102  I wanted to check in to see if you had an ETA on creating the `resource` parameter so we can merge in your PR.

Thanks
Doug

**Review by jayasimha-raghavan-unskript — APPROVED** [review]

**Pull request merged**